resources/devices/tablet/motorola.json

```diff
@@ -51,6 +64,19 @@
             "isTablet": "true"
         }
     },
+    "Motorola MZ615": {
```

Comment:
These two devices are the Wifi version of the Xoom 2 and Xoom 2 Media Edition, I did end up using one of these in these changes, the other was added for future use (perhaps).